README.md

```diff
@@ -46,4 +54,22 @@ Connect your arduino into usb port and then run:
 
 ```
 $ python run.py
+
+```
+## Possible Problems
+
+**StandardFirmata don't work**
+
+If you're having some problems with the StandardFirmata, add the SoftwareSerial header with the code inclues:
+
+```
+#include <SoftwareSerial.h>
+```
+
+**pingo-io is empty**
```

Comment:
@akarokr i find solution... when clone we need use `--recursive` when repository works with submodule. Something like that:

`$ git clone --recursive <your repository>`

This is a correct way to fix this. I add this comment on README before this PR... thanks for report this problem :smile: .

Reference: https://git-scm.com/book/en/v2/Git-Tools-Submodules